examples/classic/sequential/BinSearch/BinSearch.qnt

```diff
@@ -44,7 +44,7 @@ module BinSearch {
               -(MAX_UINT - wrapped)
 
     // Initialization step (lines 2-3)
-    action Init = all {
+    action init = all {
```

Comment:
This is really confusing... the action names below are upper case, but `init` and `next` are lower case?

I understand that you need standardized names for the runner script, but I think consistent style is important for code examples.

Reply:
Makes sense. I’ll fix the other operator names in a followup.